inventory_management_system_api/models/catalogue_item.py

```diff
@@ -12,29 +12,31 @@
 
 class PropertyIn(BaseModel):
     """
-    Model representing a catalogue item property.
+    Model representing a catalogue item property in.
     """
 
     id: CustomObjectIdField = Field(serialization_alias="_id")
     name: str
     value: Any
+    unit_id: Optional[CustomObjectIdField] = None
     unit: Optional[str] = None
 
 
 class PropertyOut(BaseModel):
     """
-    Model representing a catalogue item property.
+    Model representing a catalogue item property out.
     """
 
     id: StringObjectIdField = Field(alias="_id")
     name: str
     value: Any
+    unit_id: Optional[StringObjectIdField] = None
     unit: Optional[str] = None
 
     model_config = ConfigDict(populate_by_name=True)
 
 
-class CatalogueItemBase(BaseModel):
+class CatalogueItemBaseIn(BaseModel):
```

Comment:
```suggestion
class CatalogueItemIn(BaseModel):
```
Again I don't think we need the BaseIn and BaseOut, the properties should just be overridden in the In and Out models.